app/decorators/camaleon_cms/site_decorator.rb

```diff
@@ -127,7 +127,7 @@ def draw_languages(list_class = "language_list list-inline pull-right", current_
     lan.each do |lang|
       path = lang.to_s+'.png'
       img = "<img src='#{h.asset_path("camaleon_cms/language/#{path}")}'/>"
-      res << "<li class='#{ current_class if I18n.locale.to_s == lang.to_s}'> <a href='#{h.cama_url_to_fixed(current_page ? "url_for" : "cama_root_url", {locale: lang})}'>#{img}</a> </li>"
+      res << "<li class='#{ current_class if I18n.locale.to_s == lang.to_s}'> <a href='#{h.cama_url_to_fixed(current_page ? "url_for" : "cama_root_url", {locale: lang})}'>#{lang.to_s.upcase}</a> </li>"
```

Comment:
This change you need to do in your local project, i.e: you need to overwrite the method using an initializer.
sample: http://camaleon.tuzitio.com:8080/documentation/category/40757-developer/customize-post-type-routes.html
or
http://camaleon.tuzitio.com/documentation/category/40757-developer/customize-post-type-routes.html